cmd/node/factory/structs.go

```diff
@@ -7,7 +7,6 @@ import (
 	"encoding/hex"
```

Comment:
in core/address.go:
-> fix imports
-> lines 59-63 could be replaced with `return isOnMetaChainSCAddress`

Reply:
done